data/name-corrections/nameCorrections.csv

```diff
@@ -0,0 +1,105 @@
+Chittagong#Comilla#Debidwar#Dakshin Rajamehar,Chittagong#Comilla#Debidwar#Rajamehar
+Chittagong#Noakhali#Chatkhil#Badalkot,Chittagong#Noakhali#Chatkhil#Badalkut
+Chittagong#Noakhali#Chatkhil#Chatkhil,Chittagong#Noakhali#Chatkhil#Chatkhil Paurashava
+Chittagong#Noakhali#Chatkhil#Khilpara,Chittagong#Noakhali#Chatkhil#Khil Para
+Chittagong#Noakhali#Chatkhil#Chatkhil Pourashava,Chittagong#Noakhali#Chatkhil#Chatkhil Paurashava
+Chittagong#Noakhali#Companiganj#Basurhat Pourashova,Chittagong#Noakhali#Companiganj#Basurhat Paurashava
+Chittagong#Feni#Dagan Bhuiyan,Chittagong#Feni#Daganbhuiyan
+Chittagong#Feni#Dagan Bhuiyan#Purba Chandpur,Chittagong#Feni#Daganbhuiyan#Purba Chandrapur
+Chittagong#Feni#Dagan Bhuiyan#Pourashava,Chittagong#Feni#Daganbhuiyan#Daganbhuiyan Paurashava
+Chittagong#Feni#Feni Sadar#Kalidaha,Chittagong#Feni#Feni Sadar#Kalidah
+Chittagong#Feni#Feni Sadar#Panchgachhia,Chittagong#Feni#Feni Sadar#Panchgachhiya
+Dhaka#Faridpur#Boalmari#Boalmari Pourashava,Dhaka#Faridpur#Boalmari#Boalmari Paurashava
+Dhaka#Faridpur#Faridpur Sadar#Deceerchar,Dhaka#Faridpur#Faridpur Sadar#Decreerchar
+Dhaka#Faridpur#Faridpur Sadar#Faridpur Municipality,Dhaka#Faridpur#Faridpur Sadar#Faridpur Paurashava
+Dhaka#Narayanganj#Araihazar#Kalapaharia,Dhaka#Narayanganj#Araihazar#Kala Paharia
+Dhaka#Narayanganj#Araihazar#Uchitpura,Dhaka#Narayanganj#Araihazar#Uchitpur
+Dhaka#Shariatpur#Bhedarganj#Digar Mahishakhali,Dhaka#Shariatpur#Bhedarganj#Digar Mahishkhali
+Dhaka#Shariatpur#Damudya#Paschim Damudya,Dhaka#Shariatpur#Damudya#Purba Damudya
+Dhaka#Shariatpur#Damudya#Islampur,Dhaka#Shariatpur#Damudya#Islam Pur
+Dhaka#Manikganj#Daulatpur#Char Katari,Dhaka#Manikganj#Daulatpur#Charkatari
+Dhaka#Madaripur#Kalkini#Char Daulatkhan,Dhaka#Madaripur#Kalkini#Char Daulat Khan
+Dhaka#Madaripur#Kalkini#Kalkini Pourashava,Dhaka#Madaripur#Kalkini#Kalkini Paurashava
+Dhaka#Gopalganj#Kashiani#Hatira,Dhaka#Gopalganj#Kashiani#Hatiara
+Dhaka#Munshiganj#Lohajang#Khidirpara,Dhaka#Munshiganj#Lohajang#Khidir Para
+Dhaka#Munshiganj#Lohajang#Lohajang,Dhaka#Munshiganj#Lohajang#Lohajang Teotia
+Dhaka#Munshiganj#Lohajang#Medini Mondal,Dhaka#Munshiganj#Lohajang#Medini Mandal
+Rajshahi#Sirajganj#Belkuchi#Baradhul,Rajshahi#Sirajganj#Belkuchi#Bara Dhul
+Rajshahi#Sirajganj#Kamarkhanda#Jamtali,Rajshahi#Sirajganj#Kamarkhanda#Jamtail
+Rajshahi#Sirajganj#Kamarkhanda#Roydaulatpur,Rajshahi#Sirajganj#Kamarkhanda#Roy Daulatpur
+Rajshahi#Sirajganj#Kazipur#Chalita Danga,Rajshahi#Sirajganj#Kazipur#Chalitadanga
+Rajshahi#Sirajganj#Kazipur#Nischintapur,Rajshahi#Sirajganj#Kazipur#Nishchintapur
+Rajshahi#Sirajganj#Kazipur#Kazipur Pourasova,Rajshahi#Sirajganj#Kazipur#Kazipur Paurashava
+Rajshahi#Bogra#Dhunat#Kalerpara,Rajshahi#Bogra#Dhunat#Kaler Para
+Barisal#Bhola#Bhola,Barisal#Bhola#Bhola Sadar
+Barisal#Bhola#Lalmohan#Lord Hardinge,Barisal#Bhola#Lalmohan#Lord Hardinje
+Barisal#Bhola#Lalmohan#Lalmohan Pourashava,Barisal#Bhola#Lalmohan#Lalmohan
```

Comment:
before merging, we will want to include name corrections Mo gave to Jacek some time back – task for @jacekkopecky to dig it up